questionnaire/metadata/metadata-service.js

```diff
@@ -32,7 +32,8 @@ function createMetadataService({
                     created: data.created,
                     modified: data.modified,
                     expires: data.expires,
-                    'submission-status': data.submission_status
+                    'submission-status': data.submission_status,
+                    'analytics-id': data.analytics_id
```

Comment:
I purposefully didn't use the word "questionnaire" to stop it being conflated with the `questionnaireId`. If you think it could have a use for something other than analytics, I'm happy to revisit this.